lib/src/components/__tests__/SimpleComponents.test.tsx

```diff
@@ -117,66 +116,6 @@ describe('NxH4', function() {
   });
 });
 
-describe('NxTile', function() {
```

Comment:
These tests still need to exist, even if not in this file

Reply:
> These tests still need to exist, even if not in this file

Brought it back here: d024164426708062d22c527c0a562490c1d6a73a